CONTRIBUTING.md

```diff
@@ -18,18 +16,18 @@ Both API and implementation stability are important to Nighthawk. Since the API
 
 # Submitting a PR
 
-* Generally Nighthawk mirrors [Envoy's policy](https://github.com/envoyproxy/envoy/blob/master/CONTRIBUTING.md#pr-review-policy-for-maintainers) with respect to PR submission policy.
+* Generally Nighthawk mirrors [Envoy's policy](https://github.com/envoyproxy/envoy/blob/master/CONTRIBUTING.md#submitting-a-pr) with respect to PR submission policy.
 * Any PR that changes user-facing behavior **must** have associated documentation in [docs](docs) as
-  well as [release notes](docs/root/intro/version_history.rst). 
+  well as [release notes](docs/root/version_history.rst).
```

Comment:
We may still have a wrong file extension here (.rst vs .md).

Reply:
GitHub developer docs are MD, end user docs are RST.

Reply:
Thanks for the clarification, the comment was based on the fact that the linked file doesn't exist.

@oschaaf can we create the missing doc as necessary?

Reply:
I fixed the bad link in 63710cf9543cd470141e1317e2adda5e8bd8ba82, it was some bad copying.
The file currently is MD, I was not aware of this dev/user distinction. I punted renaming this file to #244 - I will pick this up once all the doc PR's have landed.